### PR TITLE
chore: bump tailscale to 1.14.0

### PR DESCRIPTION
**Pull request by Easen**

### CODE REVIEW

**Comment by maisem** [comment]
This is fine, but note that you won't be able to install the spk directly on DSM7. 

The capabilities we need prevent manual installation on DSM7.
https://github.com/tailscale/tailscale-synology/blob/main/src/privilege-dsm7#L11

**Comment by Easen** [comment]
I've got a local branch that has the same changes, but I couldn't get it to work.

So how do you test your changes before issuing a release?

**Comment by maisem** [comment]
Just sent #82 to be able to build sideloadable spks.

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-TAILSCALE_VERSION ?= "1.12.4"
+TAILSCALE_VERSION ?= "1.14.0"
 TAILSCALE_TRACK = "stable"
 # This needs to be monotinically increasing regardless of the TAILSCALE_VERSION
 SPK_BUILD = "006"
```
Comment (maisem) [inline Makefile:4]: can you increment this too?

**Comment by gilbreen** [comment]
If one is still running DSM 6, can 1.14.0 be installed?

**Comment by maisem** [comment]
> If one is still running DSM 6, can 1.14.0 be installed?

Yes, Tailscale is supported on both DSM6 and DSM7

**Pull request merged**